blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
+- `guid`: String; unique id
```

Comment:
I believe so. I don't think it's necessary for the React implementation but kept it in to remain consistent with the xsl implementation; can foresee other non-react scripts using that so thought it would be safest to keep it in.